tests/test_queues.py

```diff
@@ -346,5 +346,39 @@ async def test_drop_all_items():
     await deactivate_test(client)
 
 
+@pytest.mark.asyncio
+async def test_rollback_timeout():
+    max_retry = 3
+    max_minutes = 2
+    client, queue_instance = await init_test(max_retry_rollback=max_retry, max_timeout_in_queue=max_minutes)
+    with patch('aiopyrq.helpers.wait_for_synced_slaves') as slaves_mock:
+        items = [1,2,3,4]
+
+        await client.execute('lpush', queue_instance.processing_queue_name, *items)
+
+        await queue_instance.drop_all_items()
+        rolledback_items = []
+
+        time.sleep(10)
```

Comment:
This may cause too much time delay in running of tests. Can we decrease it? Or better approach is monkey-patching of the time.time() func.

Even worse is the sleeping on line 373 which is called more times...